tests/main.cpp

```diff
@@ -8,20 +9,28 @@
 #include <stack>
 
 #include <GLFW/glfw3.h>
+#include "../../tangram-es/core/include/glm/glm/glm.hpp"
```

Comment:
GLM can be installed through package managers (brew and apt-get, at least) so we can just treat it as a dependency the same as GLFW and not assume that tangram-es is nearby :)